friends/src/App.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import Friend from './Components/Friend';
+import NewFriend from './Components/NewFriend';
+
+
+class App extends Component {
+  constructor(){
+    super()
+    this.state = {
+      data:[],
+      name:'',
+      id:'',
+      age:'',
+      email:'',
+      isUpdate:false
+    }
+  }
+
+  componentDidMount(){
+    this.getAllFriends();
+  }
+
+  addFriend(friend){
+    axios.post('http://localhost:5000/friends',friend)
+    .then(response => this.getAllFriends())
+    .catch(error => console.log('error. ',error))
+  }
+  getAllFriends(){
+    axios.get('http://localhost:5000/friends')
+    .then(response => this.setState({data:response.data}))
+    .catch(error => console.log('error. ',error))
+  }
+
+  handleAddNewFriend = event =>{
+    event.preventDefault();
+    if (this.state.isUpdate){
+      let friend = this.state.data.find(el =>{
+        return el.id=this.state.id;
```

Comment:
this line seems weird, you usually don't return an assignment, was this supposed to be a comparison operator (== or ===)?